filebeat/harvester/reader/line.go

```diff
@@ -90,12 +98,22 @@ func (l *Line) Next() ([]byte, int, error) {
 	return bytes, sz, nil
 }
 
+func (r *Line) Close() error {
+	close(r.done)
+	return r.rawInput.Close()
```

Comment:
the rawInput is the file handle from harvester, right? That is (*Harvester).Close will attempt to close the file a second time?